brianlong/www.validators.app
# 180493744 stake pool stats view

#### What's this PR do?
- view for stake pools

#### How should this be manually tested?
- check /stake_accounts page
- make sure pagindation works
- check if tickers are displayed where possible

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/180493744
- https://www.pivotaltracker.com/story/show/180878590
- https://www.pivotaltracker.com/story/show/180874406

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
